packages/launchpad/src/global/GlobalPageHeader.vue

```diff
@@ -5,29 +5,58 @@
     class="min-w-200px w-80% flex-grow"
     :placeholder="t('globalPage.searchPlaceholder')"
   />
+  <input
+    ref="fileInputRef"
+    type="file"
+    class="hidden"
+    webkitdirectory
+    webkit-relative-path
+    @change="handleFileSelection"
+  >
   <Button
     :prefix-icon="IconPlus"
+    aria-controls="fileupload"
     prefix-icon-class="text-center justify-center text-lg"
     class="w-20% min-w-120px text-size-16px h-full focus-within:ring-2 focus-within:ring-offset-2 focus-within:ring-indigo-500"
-    @click="$emit('new-project')"
```

Comment:
I think I've removed two of these emits in PR's and wanted to make sure I wasn't getting rid of some vue idiom.